### DISCUSSION TITLE
Can't open Project Items

### DISCUSSION THREAD

**Discussion by jwerre** [community/community#16093] [Projects and Issues] [answered]
When you click on a project item (that isn't and Issue) it just opens a new page in the Project view. It does not open the item

**Comment by ch1bo** [comment]
Stopped working for us as well.

**Comment by ch1bo** [comment]
Feedback items about this: #16083, #16093, #16103 (at least)

**Comment by ch1bo** [comment]
Enabling the feature preview setting (again?) seems to be a solution to this! https://github.com/github/feedback/discussions/16136#discussioncomment-2699638

**Comment by egilchrist0110** [comment]
If you have custom fields on the project item and convert it into an issue, you are unable to edit those custom fields now that it is only opening it up in the issue. We use those custom fields for tracking and filtering.

**Comment by azenMatt** [comment, marked as answer]
Hi - this is now fixed, but please let me know if you are still having any problems
